src/Data/Formatter/Transformer/Entity/Drupal7.php

```diff
@@ -5,11 +5,20 @@
 use Acquia\ContentHubClient\Attribute;
 use Acquia\ContentHubClient\Data\Formatter\Transformer\General;
 
+/**
+ * Drupal 7 data transformer class.
+ */
 class Drupal7 extends General
 {
+    /**
+     * Transform - array string to string.
+     *
+     * @param mixed $data Data to transform
+     * @param mixed $index Transform data of index
+     */
     public function arrayStringToString(&$data, $index)
     {
-        if (!isset($data[$index])) {
+        if (!isset($data[$index]) || empty($data[$index]['type']) || empty($data[$index]['value'])) {
```

Comment:
@abarriosr, is this line update correct? Basically, if the entry doesn't already have 'type' and 'value', don't update it?